src/rf/apps/core/decorators.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import json
+from functools import wraps
+
+from django.http import HttpResponse, QueryDict
+from django.http.response import Http404
+
+from apps.core.exceptions import (BadRequest, Forbidden, InvalidApiCredentials,
+                                  MethodNotAllowed, Unauthorized)
+from apps.core.models import User
+
+
```

Comment:
At a glance, it doesn't look like it, but is there much overlap between this and `django-tinsel`?